config/routes.rb

```diff
@@ -6,7 +6,9 @@
 
   # Create named routes for each collection specified in tabs.yml
   YAML.load_file( File.join(Rails.root, "config", "repositories.yml") )["Catalog"]["repositories"].each do |coll|
```

Comment:
@chrpr I still think you should use the Repositories class here:

https://github.com/NYULibraries/findingaids/blob/development/app/controllers/catalog_controller.rb#L114